openshift/template.yaml

```diff
@@ -0,0 +1,242 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: configuration-anomaly-detection-template
+parameters:
+- name: IMAGE_DIGEST
+  value: sha256:22d2a957d935e883f45addd35acc87450ce71ea2f94c33f2df70ff36b015486a
```

Comment:
Question: These are only set when the parameter hasn't been set, correct?

Reply:
yes